crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -740,6 +788,7 @@ impl ExecContext {
                     (None, false) => self.manager.stop().await,
                     // Send task
                     (Some(client), true) => {
+                        // TODO don't do this
```

Comment:
is this saying not to send logs to spaces if the TUI was enabled?

Reply:
Oops, that comment is leftover from a hack that I fixed. Will remove comment since the hack is no longer there